mangdangroboticsclub/minipupper_ros
# Update for mini_pupper_navigation

## Proposed changes
I have adjusted the cartographer slam.lua configuration with @0nhc, the base frame is /base_link now.
And the map for localization is mymap.pbstream which will be stored in /home directory if the user follows the new README.

These changes will make the navigation available on a real mini pupper.
### Types of changes


## Testing and Verification
To verify these changes, I performed the following tests:

1. Run the bringup launch file on Mini Pupper.
2. Run SLAM with a keyboard.
3. Save the map.
4. Run localization.
5. Run Navigation.
for more information, check the new README.

## Test Configuration

__Mini Pupper Version__  
Mini Pupper

__Raspberry Pi OS + ROS version__  
Ubuntu 22.04, ROS 2 Humble

__PC OS + ROS version__  
Ubuntu 22.04, ROS 2 Humble

## Checklist
